site/en/posts/site-refactor-2022.md

```diff
@@ -0,0 +1,92 @@
+
+## Static Site Scaffold
+
+When ChromeOS.dev was first built, we chose [Eleventy](https://www.11ty.dev/) as our static site generator. We actually made this decision back in 2019, started building on version `0.10`, and upgraded to `0.11` before launch. At the time, there was a lot of energy around Eleventy, with lots of people making their own starter kits for it. We were no exception, and built our own: [Static Site Scaffold](https://github.com/chromeos/static-site-scaffold-modules/tree/85131342337ce51bd023e43c13161ad8ed6c4dac/modules/static-site-scaffold). Our goal was to offer the work we were doing to the community, providing a foundation for people to build internationalized Eleventy sites with, along with default configurations for Rollup and Gulp for managing assets. We also built a number of [shared modules](https://github.com/chromeos/static-site-scaffold-modules/tree/85131342337ce51bd023e43c13161ad8ed6c4dac/modules), ranging from Eleventy plugins to commit linters to markdown plugins and service worker libraries. We had high hopes that others would pick up these modules and use them themselves. For a variety of reasons, they never got the traction we were hoping for.
+
+Because of this, we decided the complexity of maintaining our configuration separately from our site (Static Site Scaffold) was no longer worthwhile, so we only kept individual pieces we had seen real need to re-use across projects. Our first set of work, then, was [migrating our Eleventy config into our codebase](https://github.com/chromeos/chromeos.dev/issues/147), and modularizing our [localization logic](https://github.com/chromeos/chromeos.dev/issues/145) and [markdown config](https://github.com/chromeos/chromeos.dev/issues/146) into separate modules. The localization logic was especially useful to modularize, as, right before our initial deploy, there was a bug in Node 12 that broke our ability to use [full-icu](https://www.npmjs.com/package/full-icu) and left us pinned on a specific Node 12 version. During the the modularization, we learned that the bug had been fixed and that, in later versions of Node, it was no longer needed at all, further simplifying our codebase.
+
+## A Shift in Tooling
+
+We launched ChromeOS.dev August of 2020. Between when we launched the site and the end of the year, a tectonic shift in development tooling happened, starting with [Snowpack 2.0](https://www.npmjs.com/package/snowpack) and [esbuild](https://www.npmjs.com/package/esbuild), then [WMR](https://www.npmjs.com/package/wmr), and finally [Vite 2.0](https://www.npmjs.com/package/vite). The bundler ecosystem embraced ES modules and a "batteries included mentality, making them lightning fast while removing the need to configure complex features, like code splitting and hot module reloading. The result? Faster development that required less maintenance and produced more optimized code. We're in.
```

Comment:
nit: The first sentence in this paragraph is not a good topic sentence. Consider something like "Between when we launched this site in August of 2020 and the end of that year, a tectonic shift in development tooling happened. 3 specific packages had excited us - [Snowpack 2.0]() ...."